src/server/main_service.cc

```diff
@@ -877,6 +877,16 @@ void Service::Init(util::AcceptServer* acceptor, std::vector<facade::Listener*>
   config_registry.RegisterMutable("dbfilename");
   config_registry.RegisterMutable("table_growth_margin");
   config_registry.RegisterMutable("pipeline_squash");
+  config_registry.RegisterMutable("pipeline_queue_limit",
+                                  [pool = &pp_](const absl::CommandLineFlag& flag) {
+                                    auto res = flag.TryGet<uint32_t>();
+                                    if (res.has_value()) {
+                                      pool->AwaitBrief([val = *res](unsigned, auto*) {
+                                        facade::Connection::SetMaxQueueLenThreadLocal(val);
+                                      });
+                                    }
+                                    return res.has_value();
+                                  });
```

Comment:
note that this won't unblock the connections if you wanted to get a frozen instance going faster as we don't call notify()

Reply:
that's fine, it was not the intent.